pkg/kev/skaffold.go

```diff
@@ -109,22 +106,78 @@ func LoadSkaffoldManifest(path string) (*SkaffoldManifest, error) {
 	return s, yaml.Unmarshal(data, &s)
 }
 
-// AddProfiles injects kev profiles to existing Skaffold manifest
+// InjectProfiles injects kev profiles to existing Skaffold manifest
```

Comment:
Not sure abou Add vs Inject, feels like They convey the same message and Add seems to be more used overall 🤔

Reply:
Agree, they both mean the same in this context. Changed to `Inject` to indicate that this is something we normally only do to an already existing Skaffold manifest, i.e. we inject additional external profiles to it which weren't there initially. The surface area of this func is small so will leave as is for now.